ExerciseScript4.3

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#This will hopefully work
```

Comment:
Probably an accidental comment? But you would not want something like this in a final pull request.